common/scaffolding/reports/a11y-2.json

```diff
@@ -0,0 +1,441 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/assessment-report-en.json-ld",
+
+  "@type": [ "earl:Assertion", "acr:AssessmentReport" ],
+
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": "https://wet-boew.github.io/GCWeb/common/scaffolding/forms.html",
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "Evaluation of the forms page whose purpose is to test all native forms related elements, if they are aligned with our design and are compliant to our accessibility guideline when used as is without any special customization.",
+    "earl:pointer": {
+      "@type": "oa:CssSelector",
+      "@value": ""
```

Comment:
Missing the CSS selector that identify what that has been tested for WCAG. Is it the whole page?